homeworks/nikita.karchinskiy_Nikkarchinskiy/6-typescript/src/index.ts

```diff
@@ -0,0 +1,69 @@
+const letterBox = document.querySelector<HTMLDivElement>('[data-letter-task]')!;
+const scoreBox = document.querySelector<HTMLDivElement>('[data-score]')!;
+const scoreDif = document.querySelector<HTMLDivElement>('[data-circle]')!;
+const mainWrapper = document.querySelector<HTMLDivElement>('[data-main-wrapper]')!;
+
+let score: number = 100;
+let letter: string;
+let timer: number;
+scoreBox.innerText = '100';
```

Comment:
What if we will need to change the default score value? Another developer and forgot about this variable, and the code will have the wrong logic. You need to use `score` here.